pandas/tests/groupby/transform/test_transform.py

```diff
@@ -1535,3 +1535,151 @@ def test_transform_sum_one_column_with_matching_labels_and_missing_labels():
     result = df.groupby(series, as_index=False).transform("sum")
     expected = DataFrame({"X": [-93203.0, -93203.0, np.nan]})
     tm.assert_frame_equal(result, expected)
+
+
+def test_min_one_unobserved_category_no_type_coercion():
+    df = DataFrame({"A": Categorical([1, 1, 2], categories=[1, 2, 3]), "B": [3, 4, 5]})
```

Comment:
Add the issue the test is being for (or PR if there is no issue), e.g. `# GH#58084`

Reply:
Resolved

Reply:
To agree with the style of the rests of the tests, can you add the `# GH#58084` in the body of the function:

```
def test_min_one_unobserved_category_no_type_coercion():
    # GH#58084
    df = ...
```